modules/govuk_env_sync/manifests/sync_script.pp

```diff
@@ -6,7 +6,14 @@
   sudo::conf {
     'govuk-env-sync-commands':
       ensure  => 'present',
-      content => 'govuk-backup ALL=(root,postgres) NOPASSWD:/usr/bin/psql,/usr/bin/createdb,/usr/bin/dropdb,/usr/bin/dropuser,/usr/bin/pg_restore,/usr/bin/mysql,/usr/bin/mysqldump,/usr/bin/pg_dump';
+      content => 'govuk-backup ALL=(root,postgres) NOPASSWD:/usr/bin/psql,/usr/bin/createdb,/usr/bin/dropdb,/usr/bin/dropuser,/usr/bin/pg_restore,/usr/bin/mysql,/usr/bin/mysqldump,/usr/bin/pg_dump,/usr/bin/docker';
```

Comment:
We talked about potentially running Docker as non-root and some other options, but again given the current state of things it'd be a lot of effort for probably no real-world improvement. This seems like the right engineering tradeoff to me given that it's not really making things worse and the immediate goal is getting working backup/restore 👍

Reply:
👍🏻  - thanks, yeah. I also thought about pulling the whole `docker run postgres pg_dump` bit out into its own script, and then allowing that script to be run with sudo (so the script would run `sudo /usr/local/bin/docker-pg-dump.sh`). I think that would be "secure" (because you couldn't get `docker-pg-dump.sh` to do anything other than dump databases), but it felt theatrical given the other stuff in sudoers.